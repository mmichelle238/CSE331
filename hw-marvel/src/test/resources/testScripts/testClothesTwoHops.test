# Find a path that that takes 2 hops to get to the destination.

LoadGraph clothes newClothes.tsv

FindPath clothes pants sweatshirts