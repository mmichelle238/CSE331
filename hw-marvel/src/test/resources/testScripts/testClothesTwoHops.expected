# Find a path that that takes 2 hops to get to the destination.

loaded graph clothes

path from pants to sweatshirts:
pants to overalls via pantbook
overalls to sweatshirts via shirtbook